src/napari_omero/plugins/loaders.py

```diff
@@ -128,3 +143,69 @@ def get_lazy_plane(zct: tuple[int, ...]):
             c_stacks.append(da.stack(z_stack))
         t_stacks.append(da.stack(c_stacks))
     return da.stack(t_stacks)
+
+
+def get_pyramid_lazy(image: ImageWrapper) -> list[da.Array]:
+    """Get a pyramid of rgb dask arrays, loading tiles from OMERO."""
+    size_z = image.getSizeZ()
+    size_t = image.getSizeT()
+    size_c = image.getSizeC()
+    pixels = image.getPrimaryPixels()
+    dtype = PIXEL_TYPES.get(pixels.getPixelsType().value, None)
+
+    image._prepareRenderingEngine()
+    tile_w, tile_h = image._re.getTileSize()
+
+    def get_tile(tile_name):
+        """tile_name is 'level,z,t,x,y,w,h'."""
+        level, z, c, t, x, y, w, h = (int(n) for n in tile_name.split(","))
+        pix_id = image.getPixelsId()
+        with raw_pixels_store(image) as pix:
+            pix.setPixelsId(pix_id, False)
```

Comment:
Possibly here, we need to pass a cross-group context as a 3rd argument to `setPixelsId()` as at https://github.com/ome/omero-py/blob/05d52412ff7d6c4e3a515717d2807b1a21bf4291/src/omero/gateway/__init__.py#L7422

```
pix.setPixelsId(pix_id, False, {"group": "-1"))
```

Reply:
Alas, that did not fix it.
I wonder if it's related to:
https://github.com/tlambert03/napari-omero/issues/86
When using Groups: to select All, I get `    message = No such group: -1` (as admin again)

Reply:
Hmmm - I'm not sure... Do you have to select "All" groups to see this error? I thought you were seeing it when you simply select any group that's not your "default" group?

Reply:
to clarify:
If I access an NDPI within my Group (mine or another user does't matter), it loads multiscale just fine.
If I select Groups: All I get #86 , but it lets me set User: All and see all the projects.
Then when I select the same NDPI, pixels never load and if I do `.compute()` on the dask array I get the 
`serverStackTrace = ome.conditions.ValidationException: Cannot read pixels id=417758` error.

Reply:
@will-moore your hint gave me enough to google and I got it to work. It's `"omero.group": "-1"`
Just pushed the commit.